tests/lib.rs

```diff
@@ -21,7 +21,15 @@
 
 extern crate self_encryption;
 extern crate rand;
+extern crate rustc_back;
```

Comment:
Why this instead of http://web.mit.edu/rust-lang_v1.0al/arch/amd64_ubuntu1404/share/doc/rust/html/std/io/struct.TempDir.html